lzy/scheduler/src/main/java/ai/lzy/scheduler/allocator/AllocatorImpl.java

```diff
@@ -102,30 +65,17 @@ public void shutdown() {
     }
 
     @Override
-    public void allocate(String userId, String workflowName, String workerId, Operation.Requirements requirements) {
-        String privateKey;
-        try {
-            var workerKeys = RsaUtils.generateRsaKeys();
-            privateKey = workerKeys.privateKey();
-
-            final var subj = subjectClient.createSubject(AuthProvider.INTERNAL, workerId, SubjectType.WORKER,
-                new SubjectCredentials("main", workerKeys.publicKey(), CredentialsType.PUBLIC_KEY));
+    public String createSession(String userId, String workflowName, String idempotencyKey) {
```

Comment:
we already have allocator session at LzyService (after spawning of the portal)
let's reuse it for workers too

Reply:
no, single session is enough

Reply:
Ok, will do it in other PR